_posts/2018-02-15-upgrading-a-monolith.markdown

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Upgrading a monolith from Rails 4.0 to 5.1"
+date: 2018-02-15
+categories: ["rails", "upgrades", "case-study"]
+author: "emily"
+---
+
+We recently completed work with [Power Home Remodeling](https://powerhrg.com) on a Rails upgrade for their self-described “monolith CRM/ERP application”, and were able to speak to them about their experience with Ombu Labs.
```

Comment:
@ercohen14 Maybe this sounds better "We recently collaborated with..."? 

Please link `Rails upgrade` to `https://fastruby.io`

Please link `Ombu Labs` to `https://www.ombulabs.com`